packages/core/src/templates/ResourceNotes.ts

```diff
@@ -20,7 +20,7 @@ export interface NoteSchema extends Record<string, unknown> {
   updated_at: string;
   system: boolean;
   noteable_id: number;
-  noteable_type: 'Issue' | 'Snippet' | 'Epic' | 'Commit' | 'Merge request';
+  noteable_type: 'Issue' | 'Snippet' | 'Epic' | 'Commit' | 'MergeRequest';
```

Comment:
https://docs.gitlab.com/ee/api/discussions.html#list-project-merge-request-discussion-items

According to the doc, it can be `Merge request` unfortunately. Bad design.

Reply:
Is it possibly a typo? Ill dig into the source to verify

Reply:
Looks like just  a typo in the docs: https://gitlab.com/gitlab-org/gitlab/-/blob/master/app/models/note.rb

Reply:
OK then... 😅

Reply:
I raised https://gitlab.com/gitlab-org/gitlab/-/merge_requests/140293 to fix the typos.